infrastructure/models.py

```diff
@@ -23,6 +23,22 @@ class Meta:
     def __str__(self):
         return self.name
 
+class OwnerStake(models.Model):
+    """ Percentage that owners own and also relation with PowerPlant and Organization """
+    power_plant = models.ForeignKey(
+        'PowerPlant',
+        models.CASCADE,
+        related_name = 'owner_stake'
```

Comment:
Can we change this `related_name` to `owner_stakes`, since the `PowerPlant` can have many owner stakes?